lnrpc/lightning.proto

```diff
@@ -3750,10 +3750,23 @@ message Payment {
     string payment_request = 9;
 
     enum PaymentStatus {
-        UNKNOWN = 0;
+        // Deprecated.
+        UNKNOWN = 0 [deprecated = true];
```

Comment:
Add comment that it is not only deprecated, but also never returned?

Reply:
added.